src/helpers/figures/charts/chart_factory.ts

```diff
@@ -184,6 +187,18 @@ export function getSmartChartDefinition(zone: Zone, getters: Getters): ChartDefi
       legendPosition: newLegendPos,
     };
   }
+  const _dataSets = createDataSets(getters, dataSets, sheetId, dataSetsHaveTitle);
```

Comment:
The dataset is built with `dataSetsHaveTitle` being `true` even though it's forced to `false` below. Can't it be an issue?

Reply:
I don't think so, as the `fullZone` will be the same with `dataSetsHaveTitle` set to `false` or `true`. But if you prefer, I can recall `createDataSets` in the `if` below ?